src/coreclr/tools/Common/Compiler/VectorFieldLayoutAlgorithm.cs

```diff
@@ -44,9 +44,28 @@ public override ComputedInstanceFieldLayout ComputeInstanceLayout(DefType defTyp
                     alignment = new LayoutInt(16);
                 }
             }
+            else if (name == "Vector256`1")
```

Comment:
Please don't make changes like this without testing that these changes have the expected impact on field layout

Reply:
Added tests mirroring the existing Vector128 and int8x16x2 tests.